aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md

```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
 1. Log in to the [Azure Portal](https://go.microsoft.com/fwlink/?linkid=529715&amp;clcid=0x409).
 2. Click **NEW** at the bottom of the page, and then select **STORE**:
 
-    [![Displays menu with STORE entry selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.png)
+    [![Screenshot of Azure Portal menu with STORE entry selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.png)
 3. In the **Choose and Add-on** wizard, select **ClearDB MySQL Database**, and then click the **Next** arrow at the bottom of the frame:
 
    [Click the following image to expand it. ]
-    [![Displays Choose and Add-on wizard, with ClearDB MySQL Database selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image4.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image3.png)
+    [![Screenshot of Choose and Add-on wizard, with Clear D B My S Q L Database selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image4.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image3.png)
```

Comment:
A little too short for recommended specs and a typo; consider: "Screenshot of the Choose an Add-on wizard, with Clear D B My S Q L Database highlighted with a red rectangle."